x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/view.tsx

```diff
@@ -171,7 +169,7 @@ export class ServiceIntegrationsView extends React.Component<
           isOpen={this.state.activeFlyout === 'ML'}
           onClose={this.closeFlyouts}
           urlParams={this.props.urlParams}
-          serviceTransactionTypes={this.props.serviceDetails.types}
+          serviceTransactionTypes={this.props.transactionTypes}
```

Comment:
@jasonrhodes Would you mind me renaming `serviceTransactionTypes` to just `transactionTypes` to align it with [other places](https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/public/components/app/ServiceDetails/view.tsx#L51).